pkg/types/convert_test.go

```diff
@@ -271,7 +271,7 @@ func TestConvertType(t *testing.T) {
 	require.Equal(t, int64(2015), v)
 	v, err = Convert(ZeroDuration, ft)
 	require.NoError(t, err)
-	require.Equal(t, int64(0), v)
+	require.Equal(t, int64(time.Now().Year()), v)
```

Comment:
This test may fail when line 272 executes in the past year than this line. But it's ok :)

Reply:
Yes. Once TiDB fixes https://github.com/pingcap/tidb/issues/31648, we can make these tests stabler.